tests/api/fixtures/items.js

```diff
@@ -48,19 +32,27 @@ const API = module.exports = {
   },
 
   createRandomizedItems: (user, itemsData) => {
-    return API.createItems(user, itemsData.map(randomizedItem))
+    return API.createItems(user, itemsData.map(fillItemWithRandomData))
   }
 }
 
-const randomizedItem = itemData => {
+const transactions = [ 'giving', 'lending', 'selling', 'inventorying' ]
+const listings = [ 'private', 'network', 'public' ]
+
+const fillItemWithRandomData = (itemData = {}) => {
   itemData.listing = itemData.listing || _.sample(listings)
   itemData.transaction = itemData.transaction || _.sample(transactions)
   itemData.details = faker.hacker.phrase()
   itemData.notes = faker.lorem.paragraph()
   return itemData
 }
 
-const addDefaultEntity = entityUri => item => {
-  if (!item.entity) item.entity = entityUri
-  return item
+const addDefaultEntity = async itemData => {
+  if (!itemData.entity) {
+    const entity = itemData[0] && itemData[0].entity
+    const entityUriPromise = entity ? Promise.resolve(entity) : getEditionUri()
+    const entityUri = await entityUriPromise
```

Comment:
it's ok to `await` objects that aren't promises, the result is equivalent to `Promise.resolve(obj)` :D
```suggestion
    const entityUri = await (entity ? entity : getEditionUri())
```
recommendation level: suggestion